package.json

```diff
@@ -1,47 +1,30 @@
 {
-  "name": "werewolf-virtual-cards",
-  "version": "0.1.0",
-  "private": true,
-  "devDependencies": {
-    "babel-eslint": "^8.2.6",
-    "babel-plugin-inline-dotenv": "^1.1.2",
-    "eslint": "^5.1.0",
-    "eslint-config-airbnb": "^17.0.0",
-    "eslint-config-airbnb-base": "^13.0.0",
-    "eslint-plugin-import": "^2.13.0",
-    "eslint-plugin-jsx-a11y": "^6.1.1",
-    "eslint-plugin-react": "^7.10.0",
-    "eslint-plugin-react-native": "^3.2.1",
-    "jest-expo": "~27.0.0",
-    "prettier-eslint": "^8.8.2",
-    "react-native-scripts": "1.14.0",
-    "react-test-renderer": "16.3.1"
-  },
-  "main": "./node_modules/react-native-scripts/build/bin/crna-entry.js",
+  "main": "node_modules/expo/AppEntry.js",
   "scripts": {
-    "start": "react-native-scripts start",
-    "eject": "react-native-scripts eject",
-    "android": "react-native-scripts android",
-    "ios": "react-native-scripts ios",
-    "test": "jest"
-  },
-  "jest": {
-    "preset": "jest-expo"
+    "start": "expo start",
+    "android": "expo start --android",
+    "ios": "expo start --ios",
+    "eject": "expo eject"
   },
   "dependencies": {
-    "@expo/vector-icons": "^6.3.1",
-    "expo": "27.1.1",
-    "firebase": "^5.2.0",
-    "native-base": "^2.4.4",
-    "prop-types": "^15.6.2",
-    "react": "16.3.1",
-    "react-native": "~0.55.2",
-    "react-native-animatable": "^1.3.0",
-    "react-native-qrcode": "^0.2.6",
-    "react-navigation": "^3.0.9",
-    "react-redux": "^5.0.7",
-    "redux": "^4.0.0",
+    "@expo/vector-icons": "^9.0.0",
+    "expo": "^32.0.0",
+    "firebase": "^5.8.3",
+    "native-base": "^2.11.0",
+    "prop-types": "^15.7.2",
+    "react": "16.5.0",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-32.0.0.tar.gz",
```

Comment:
Why are you using a tar.gz version of react native instead of the npm package?

Reply:
I'm not sure why Expo use a tar.gz for its React Native in the [upgrading walkthrough](https://docs.expo.io/versions/latest/workflow/upgrading-expo-sdk-walkthrough/) instead of the React Native package from Facebook. Later I'm testing with the Facebook's React Native 0.57 which is the base of SDK32 according to [this post](https://blog.expo.io/expo-sdk-v32-0-0-is-now-available-6b78f92a6c52), It would be a great exercise to compare the Expo's React Native with the Facebook.